nttt/cleanup_markdown.py

```diff
@@ -1,11 +1,17 @@
 import re
+from .nttt_logging import nttt_display_md
 
 
-def trim_md_tags(file_content):
-    return re.sub(r'(?P<tag>`|_{1,3}|\*{1,3})(?P<content>.+?)(?P=tag)', replacement_builder, file_content)
+def trim_md_tags(file_content, logging="off"):
+    return re.sub(r'(?P<tag>`|_{1,3}|\*{1,3})(?P<content>.+?)(?P=tag)', replacement_builder(logging), file_content)
 
 
-def replacement_builder(matchobj):
-    tag_name = matchobj.group("tag")
-    stripped_content = matchobj.group("content").strip()
-    return "{}{}{}".format(tag_name, stripped_content, tag_name)
\ No newline at end of file
+def replacement_builder(logging):
```

Comment:
Using currying technique here to avoid code duplication

Reply:
I transferred currying technique to tag_trimming module